tests/conftest.py

```diff
@@ -4,6 +4,7 @@
 
 from core.models import Event, User
 from pictures.models import StockPicture
+from sponsor.models import Donor, Quote
```

Comment:
Tests are breaking because there is a missing migration file. If you run `./manage.py makemigrations` it should add Donor model.